### ISSUE TITLE
can not sign out from vmware client

### ISSUE THREAD

**Issue by hristo1985** [The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool#29]
Hello,

after applying the optimization i can not sign out properly from vmware horizon session from horizon client. When i try to connect i get - The view agent reports that this desktop is currently logging off a previous session. Please try again

on console i get Task Host Window, Task Host is stopping background tasks

can you help me to find what cause this (service, scheduled task). When optimization is not applied this issue is not precent

10x

**Comment by hristo1985** [comment]
![Screenshot 2020-12-01 at 11 04 48](https://user-images.githubusercontent.com/74973504/100719626-89e51e00-33c5-11eb-9474-281d1f47baec.png)
i attach screenshot from error

**Comment by JaredKMiller** [comment]
We are intermittently seeing this same behavior with stopping "Synchronize Language Settings" on a new WVD deployment. Didn't originally attribute it to the optimization tool, but we are using it. Has there been any development on what could be causing this? Thank you.

**Comment by Altech-DR** [comment]
I'm seeing this exact issue on Windows 10 Enterprise 2004 on a Hyper-V Virtual Desktop Collection. After logoff via either command line or UI, it's getting stuck on the same portion and it seems to be preventing the hypervisor from rolling back the image. It's a pooled virtual desktop setup. I'm not seeing this with a freshly installed copy of Windows, only after implementing the optimization script.

**Comment by foocnl** [comment]
> We are intermittently seeing this same behavior with stopping "Synchronize Language Settings" on a new WVD deployment (not using VMWare). Didn't originally attribute it to the optimization tool, but we are using it. Has there been any development on what could be causing this? Thank you.

I see the same error when logging out on a new wvd deployment: "Synchronize Language Settings"
Did u manage to fix this?

**Comment by robertsteeghs** [comment]
I have the same problem on different environments running WVD (pooled & personal) after running the optimization tool, this is  caused by disabling the synchronization of language preferences. 
Maybe it's an option to disable the scheduled task that is responsible for the synchronization or do not disable Sync Settings:

Computer
SOFTWARE\Policies\Microsoft\Windows\SettingSync
DisableSettingSync
DWORD:2

**Comment by sandude-ms** [comment]
Hello,

Could you try setting the task "SpeechModelDownloadTask" back to enabled and see if that resolves the issue?  Here is the path to that task:

Task Scheduler\Task Scheduler Library\Microsoft\Windows\Speech

If that resolves it, I can make a note in the Readme, revert it to default, or both.

Thanks,

Robert M. Smith

**Comment by sandude-ms** [comment]
Closing as duplicate of issue #46
